syntax = "proto3";

service TaskMenu {
    rpc DoTask(TaskRequest) returns (TaskReply) {}
    rpc DeleteTask(TaskRequest) returns (stream TaskReply) {}
    rpc OvewriteTask(stream TaskRequest) returns (stream TaskReply) {}
}

message TaskRequest {
    string task_name = 1;
}

message TaskReply {
    string task_name = 1;
    string task_number = 2;
}